.codesandbox/ci.json

```diff
@@ -2,16 +2,16 @@
   "buildCommand": "build:codesandbox",
   "node": "12",
   "packages": [
-    "packages/x-license",
-    "packages/grid/data-grid",
-    "packages/grid/x-grid",
-    "packages/grid/x-grid-data-generator"
+    "packages/x-license-pro",
+    "packages/grid/x-data-grid",
```

Comment:
Killing `/grid/` next would be 👌 . As a side note, in Core, we include the full name of the npm package: (adding `mui-)